test/core/multi-memory/memory_copy1.wast

```diff
@@ -1 +1,40 @@
-;; TODO: test copy across different memories.
+;; test memory.copy across different memories.
+(module
+  (memory $mem0 (data "\ff\11\44\ee"))
+  (memory $mem1 (data "\ee\22\55\ff"))
+  (memory $mem2 (data "\dd\33\66\00"))
```

Comment:
What are mem1 and mem2 there for?

Reply:
they are copy-and-paste from memory_copy0.wast. they are unused there too.